Update README.md (#6597)
umbraco/UmbracoDocs: umbraco-cloud/set-up/new-shop/README.md

<figure><img src="../../.gitbook/assets/image (37).png" alt="Select Payment Methods"><figcaption><p>Select Payment Methods</p></figcaption></figure>

## Removing credit cards

> Once in a while, a credit card will be changed and therefore, there might also be a need to change this in the organization as well. 

Reviewer: ```suggestion
In some cases, you might need to change the credit card information on the Umbraco Cloud Organization. 
```

> Sometimes, it is not possible to remove a credit card from the organization right away. The reason for this is that it needs to be removed from the project, as well. In order to do  this, you will need to visit the payment section of the project which you can find through the following URL: 

Reviewer: ```suggestion
Sometimes, it is not possible to remove a credit card from the organization right away. This is because it needs to be removed from the project first. To do this, you need to visit the payment section of the project which you can find through the following URL: 

```
